std/range/package.d

```diff
@@ -4416,6 +4416,31 @@ body
     assert(approxEqual(rf, [0.0, 0.1, 0.2, 0.3, 0.4]));
 }
 
+///
+@safe unittest
+{
+    import std.stdio;
+
+    /* The following three groups all produce the same output of:
+     * 1 2 3 4 5
+     */
+
+    foreach (i; 0..5)
+        writef("%s ", i);
+    writeln();
+
+
+    import std.range : iota;
+    foreach (i; iota(0, 5))
+        writef("%s ", i);
+    writeln();
+
+    import std.algorithm : map, copy;
+    import std.conv : to;
+    iota(0, 5).map!(x => to!string(x) ~ " ").copy(stdout.lockingTextWriter());
```

Comment:
And btw, your purported output is wrong, `iota` iterates from 0 to one less than the ending value, not 1 to the ending value inclusive.